api/V1/Device.js

```diff
@@ -92,22 +77,22 @@ module.exports = function(app, baseUrl) {
    * @apiUse V1ResponseError
    */
   app.get(
-      apiUrl,
-      passport.authenticate(['jwt'], {session: false}),
-      async (request, response) => {
-          log.info(request.method + " Request: " + request.url);
-
-          // Check that the request was authenticated by a User.
-          if (!requestUtil.isFromAUser(request))
-            return responseUtil.notFromAUser(response);
+    apiUrl, [
+      check('Authorization').custom((...args) => {
+        return util.authenticate('user', ...args);
+      })
```

Comment:
Presumably we can wrap this up into a function so that we have one helper for APIs that require users and one for APIs that require devices.